INSTALL.md

```diff
@@ -71,7 +82,7 @@ Other Fedora versions or distributions may require some adaptation.
    ```bash
    alias subscription-manager="sudo \
        PYTHONPATH=/path/to/subscription-manager/src \
-       $(which python3) \
+       ~/.venvs/subscription-manager/bin/python3 \
```

Comment:
I wouldn't do these, it's up to everyone to rewrite the path to something that matches their system.

Reply:
This is the path we suggest by

```bash
mkdir -p ~/.venvs/
python3 -m venv --system-site-packages ~/.venvs/subscription-manager
```

The proble with `$(which python3)` is that it wouldn’t work without activating the virtual environment. Like that, it would make more sense to add the alias to the venv activation script rather than adding a global one.

I’ll put the `$(which python3)` back.